js/collections/UsersShort.js

```diff
@@ -0,0 +1,31 @@
+/* use this collection for any list of users that can be fetched as an array of guids */
+
+import { Collection } from 'backbone';
+import UserShort from '../models/UserShort';
+import app from '../app';
+
+
+module.exports = Collection.extend({
+  /* we have to use the older style for this collection, the ES6 style creates a bug where models
+  cannot be removed using their ids */
+
+  initialize: function (models, options) {  // eslint-disable-line object-shorthand
+    if (!options.type) {
+      throw new Error('You must provide a type to the collection');
+    }
+
+    this.url = app.getServerUrl(options.guid === app.profile.id || !options.guid ?
+      `ob/${options.type}` : `ipns/${options.guid}/${options.type}`);
+  },
+
+  model: UserShort,
+
+  parse: function (response) {   // eslint-disable-line object-shorthand
+    response = response.map((guid) => {   // eslint-disable-line no-param-reassign
```

Comment:
If instead of saying `response = ...`, you just returned the results of the map call, then you wouldn't need the 'no-param-reassign' override.